hls-test-utils/src/Test/Hls.hs

```diff
@@ -419,22 +435,21 @@ runSessionWithServerInTmpDir' plugins conf sessConf caps tree act = runSessionWi
 --
 -- Note: cwd will be shifted into a temporary directory in @Session a@
 runSessionWithServerInTmpDirCont ::
+    Pretty b =>
+    -- | whether we disable the kick action or not
+    Bool ->
```

Comment:
This looks a little bit ad-hoc, perhaps we might want a full `TestConfig` record in the future?
Just a thought, let's leave it as is, it is easy to refactor later.

Reply:
Yes, I am thinking about it too. perhaps tucking in things like `Config, SessionConfig, ClientCapabilities` into the one `TestConfig` too.